### PR TITLE
Add "+" & "x" icon toggle

### PR DESCRIPTION
**Pull request by denizulku**
I experimented with animations and decided to toggle my "+" button to also work as a cancel "x" button. I decided on animating with props. Let me know if there happens to be a better or cleaner solution. Thank you :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +14,11 @@
     "react-dom": "^16.13.1",
     "react-hook-form": "^6.8.3",
     "react-input-color": "^3.0.1",
+    "react-native-anchor-point": "^1.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
     "react-select": "^3.1.0",
+    "react-spring": "^8.0.27",
```
Comment (denizulku) [inline package.json]: Please ignore both highlighted dependencies, just realised that I don't use them at all :D

File: cypress/integration/Form.spec.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 context('Create Habit', () => {
   beforeEach(() => {
-    cy.visit('/Create')
+    cy.visit('http://localhost:3000//Create')
```
Comment (carolinwagner) [inline cypress/integration/Form.spec.js]:
```suggestion
    cy.visit('http://localhost:3000/Create')
```

File: src/components/Navigation/Navigation.js [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
-      <NavLink data-cy="create" to="/Create">
-        <AddIcon />
-      </NavLink>
-      <NavLink to="/Overview">
-        <IconMonthlyOverview />
-      </NavLink>
-    </NavigationStyled>
+    <>
+      <NavigationStyled>
+        <NavLink exact to="/" activeStyle={{ fill: '#ffffff' }}>
```
Comment (carolinwagner) [inline src/components/Navigation/Navigation.js]: you could add css variables to make your code more readable :-)

File: src/components/Navigation/Navigation.js [hunk 4]
```diff
@@ -35,9 +39,22 @@ const NavigationStyled = styled.div`
   background: #bcbbbc;
 `
 
+const HomeIconStyled = styled(IconHome)`
+  z-index: 100;
+`
+
+const OverviewIconStyled = styled(IconMonthlyOverview)`
+  z-index: 100;
```
Comment (carolinwagner) [inline src/components/Navigation/Navigation.js]: Do you really need such a high z-index? Wouldn't a lower number do the trick?

File: src/components/Navigation/Navigation.js [hunk 5]
```diff
@@ -35,9 +39,22 @@ const NavigationStyled = styled.div`
...
+`
+
 const AddIcon = styled(IconAdd)`
   z-index: 100;
   position: relative;
   bottom: 30px;
   filter: drop-shadow(1px 1px 5px rgba(0, 0, 0, 0.3));
+  border-radius: 50%;
+  background-color: ${(props) => (props.createPath ? '#ffb9b9' : '#fff')};
```
Comment (carolinwagner) [inline src/components/Navigation/Navigation.js]:
```suggestion
  background-color: ${(props) => (props.createPath ? '#ffb9b9' : '#ffffff')};
```
In your `src/components/Navigation/Navigation.js`you are using #ffffff. I guess you shouldn't swith to #fff even if it is the same color ;-) (But also here: maybe you can add css variables ;-))

**Review by carolinwagner — APPROVED** [review]
Looks very good to me :-) I simply added some minor comments

**Pull request merged**